### PR TITLE
Devices families discrimination

### PR DESCRIPTION
**Pull request by Cdriko**
I fixed the problem of .ecc file and fields names, according families of devices .
Working code with a VSP300 . Need a ty with a VMP3 family device.

### CODE REVIEW

File: easy_biologic/lib/ec_lib.py [hunk 1]
```diff
@@ -1607,5 +1657,17 @@ def technique_file( technique, device = None ):
...
         # append file type extenstion if needed
         technique += file_type
-
+    
     return technique.lower()
+
+
+def isInSP300Family(device):
+    """
+    Tel if the device is in the SP300 (VMP_300) family or not
```
Comment (bicarlsen) [inline easy_biologic/lib/ec_lib.py:1666]: Tel -> Tell. Add period at end of sentence.

File: easy_biologic/lib/ec_lib.py [hunk 2]
```diff
@@ -394,6 +394,54 @@ class DataInfo( c.Structure ):
...
+    "MPG210",
+    "MPG220",
+    "MPG240",
+    "VMP3E",
+    "VSP3E",
+    "SP50E",
+    "SP150E",
+)
+
+#------------------------------------------------------------------------------#
```
Comment (bicarlsen) [inline easy_biologic/lib/ec_lib.py:426]: Please remove.

**Comment by Cdriko** [comment]
I think it's ok now.

**Comment by Cdriko** [comment]
I think it's kleen now...

**Pull request merged**